cmd/config-current.go

```diff
@@ -253,6 +253,17 @@ func (s *serverConfig) Validate() error {
 	return nil
 }
 
+// SetCompressionConfig sets the current compression config
+func (s *serverConfig) SetCompressionConfig(extensions []string, mimeTypes []string) {
+	s.Compression.Extensions = extensions
+	s.Compression.MimeTypes = mimeTypes
+}
+
+// GetCompressionConfig gets the current compression config
+func (s *serverConfig) GetCompressionConfig() CompressionConfig {
+	return s.Compression
+}
+
```

Comment:
Validate the globals before returning the value, like here https://github.com/minio/minio/pull/6292/files#diff-142169b04329c700945c603c3042f506R161

Reply:
okay ! Thanks